ScoutSuite/output/data/inc-scoutsuite/css/scoutsuite.css

```diff
@@ -350,4 +369,131 @@ input:checked + .slider:before {
     #container-user {
         margin-top: 5rem !important;
     }
+}
+
+/* ------------------ Oliver CSS -------------------- */
+
+#account_id {
+    font-weight: bold;
+}
+
+#account_id .fa-cloud {
+    color: orange;
```

Comment:
Better keep that white, orange would hint to AWS and look a bit weird for GCP/Azure.